src/app/pages/storage/volumes/datasets/dataset-form/dataset-form.component.ts

```diff
@@ -492,7 +492,7 @@ export class DatasetFormComponent implements Formconfiguration{
         if (res_number < recommended_size_number){
           _.find(this.fieldConfig, {name:'recordsize'}).warnings = `
           Recommended record size based on pool topology: ${this.recommended_size}.
-          Other sizes could reduce sequential I/O performance and space efficiency.`
+          Lower recordsize could reduce sequential I/O performance and space efficiency.`
```

Comment:
Numerically, "smaller" is a better word.  Slightly reworded:
`A smaller record size can reduce...`